lib/nav/nav-menu.vue

```diff
@@ -141,65 +147,117 @@
         return Object.keys(_.get(window, 'kiln.navButtons', {}));
       },
       displayNavMenu() {
-        return !!_.get(this.$store, 'state.ui.currentNav');
+        const navShouldBeOpen = this.menuOptions.findIndex( (option) => option.id === this.currentDrawer ) >= 0 || this.customButtons.find((button) => button === this.currentDrawer);
```

Comment:
You can abstract the condition as you did with `!this.navBackgroundIsOpen`

Reply:
done.